libraries/freertos_plus/aws/ota/src/aws_iot_ota_agent.c

```diff
@@ -2399,6 +2404,42 @@ static void prvAgentShutdownCleanup( void )
     }
 }
 
+/*
+  * Handle any events that were unexpected in the current state.
+  */
+static void prvHandleUnexpectedEvents( OTA_EventMsg_t * pxEventMsg )
+{
+    DEFINE_OTA_METHOD_NAME( "prvHandleUnexpectedEvents" );
```

Comment:
Consider `configASSERT( pxEventMsg );`

Reply:
Changed in https://github.com/aws/amazon-freertos/pull/2025/commits/ec0eaa30008d145e3809864d7f6ce6f06f3b6f0a

Reply:
nice